pom.xml

```diff
@@ -0,0 +1,163 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Sorry to be a pain, could we follow our indentation standards of 2 spaces instead of 4.

Reply:
addressed in df34d9a